lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -140,6 +144,34 @@ function findDifference(path, actual, expected) {
       continue;
     }
 
+    if (key === '_excludes') {
+      // Re-use state from `_includes` check, if there was one.
+      /** @type {any[]} */
+      const arrToCheckAgainst = inclExclCopy || actual;
+
+      if (!Array.isArray(expectedValue)) throw new Error('Array subset must be array');
+      if (!Array.isArray(actual)) continue;
+
+      const unexpectedValues = expectedValue;
+      console.log({arrToCheckAgainst});
+      for (const unexpectedEntry of unexpectedValues) {
+        const matchingIndex = arrToCheckAgainst.findIndex(actualEntry =>
```

Comment:
We don't do any splicing, can probably just use `arrToCheckAgainst.find`

Reply:
hm... but then there's the case of `undefined` being a valid value in these arrays :P

Reply:
Ah ya, index works fine then